apps/docs/components/ui/header/header.css

```diff
@@ -57,7 +57,8 @@
 .hd-header-shadow-block {
     position: relative;
     border-radius: var(--hd-space-7);
-    filter: blur(216px);
+    filter: blur(1rem);
```

Comment:
This would render the shadow blocks not blurry enough.

Reply:
good catch